GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs

```diff
@@ -54,7 +54,13 @@ public TableSubject CurrentSubject
 
         public NotificationsViewModel Notifications { get; private set; }
         public ExamplesPanelViewModel ExamplesViewModel { get; private set; }
-        public LevelerViewModel LevelerViewModel { get; private set; }
+
+        private LevelerViewModel _levelerViewModel;
+        public LevelerViewModel LevelerViewModel
+        {
+            get => _levelerViewModel;
+            set => SetProperty(ref _levelerViewModel, value);
```

Comment:
What's the relationship between the private and the public LevelerViewModels?

Reply:
This is a common setup for the MVVM pattern to update a property through the 'INotifyPropertyChanged' interface, which the `ViewModelBase` inherits from. This is necessary so the `ClassificationPanel` view using this view model will get an updated `LevelerViewModel` whenever the property is changed, which happens on this line https://github.com/zooniverse/galaxy-zoo-touch-table/blob/master/GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs#L239 to reset the leveler counts.